dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/propertyeditor/StructuredContentEditor.java

```diff
@@ -0,0 +1,52 @@
+package de.digitalcollections.cudami.admin.propertyeditor;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.digitalcollections.model.text.StructuredContent;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.beans.PropertyEditorSupport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@SuppressFBWarnings
+public class StructuredContentEditor extends PropertyEditorSupport {
```

Comment:
Das läßt sich generalisieren, als 
`public class Editor<O extends Object> extends PropertyEditorSupport`
und man müßte nur für die `setAsText` die Typklasse ermitteln. Das passiert im Konstruktur per `this.genericType = (Class<O>) GenericTypeResolver.resolveTypeArgument(getClass(), Editor.class);`

(ungetestet, compiliert aber)